db/seeds.rb

```diff
@@ -7,14 +7,24 @@
 Pet.destroy_all
 Applicant.destroy_all
 
-@shelter_1 = Shelter.create(name: 'Aurora shelter', city: 'Aurora, CO', foster_program: false, rank: 9)
-@shelter_2 = Shelter.create(name: 'RGV animal shelter', city: 'Harlingen, TX', foster_program: false, rank: 5)
-@shelter_3 = Shelter.create(name: 'Fancy pets of Colorado', city: 'Denver, CO', foster_program: true, rank: 10)
+@shelter_1 = Shelter.create!(name: 'Aurora shelter', city: 'Aurora, CO', foster_program: false, rank: 9)
+@shelter_2 = Shelter.create!(name: 'RGV animal shelter', city: 'Harlingen, TX', foster_program: false, rank: 5)
+@shelter_3 = Shelter.create!(name: 'Fancy pets of Colorado', city: 'Denver, CO', foster_program: true, rank: 10)
+
+@applicant = Applicant.create!(first_name: 'John', last_name: 'Dough', street_address: '123 Fake Street', city: 'Denver',
+  state: 'CO', zip: 80205, description: "I'm awesome", status: 'pending')
+@applicant_1 = Applicant.create!(first_name: 'Jimmy', last_name: 'Dough', street_address: '567 Fake Street', city: 'Denver',
+  state: 'CO', zip: 80205, description: "I'm awesome", status: 'Pending')
+@applicant_2 = Applicant.create!(first_name: 'Johnny', last_name: 'Johnson', street_address: '17 Psuedo Street', city: 'Denver',
+  state: 'CO', zip: 80205, description: "I'm awesome", status: 'In Progress')
+@applicant_3= Applicant.create!(first_name: 'Timmy', last_name: 'Thompson', street_address: '57 Real Street', city: 'Denver',
+  state: 'CO', zip: 80207, description: "I'm not so awesome", status: 'Pending')
+@applicant_4= Applicant.create!(first_name: 'Tommy', last_name: 'Timson', street_address: '12227 Real Blvd', city: 'Colorado Springs',
+  state: 'CO', zip: 80909, description: "I'm the worst", status: 'Approved')
 
```

Comment:
If only we would have remembered to add these applicants right away in our model's test, that probably would have knocked 30 min of work time off of figuring out why we had an empty array. lol  😅